src/autolabel/transforms/ocr.py

```diff
@@ -0,0 +1,109 @@
+from typing import Dict, Any, List
+
+from autolabel.transforms.schema import TransformType
+from autolabel.transforms import BaseTransform
+from autolabel.cache import BaseCache
+
+
+class OCRTransform(BaseTransform):
+    """This class is used to extract text from any document using OCR. The output columns dictionary for this class should include the keys 'content_column' and 'metadata_column'
+
+    This transform supports the following image formats: PDF, PNG, JPEG, TIFF, JPEG 2000, GIF, WebP, BMP, and PNM
+    """
+
+    COLUMN_NAMES = [
+        "content_column",
+        "metadata_column",
+    ]
+
+    def __init__(
+        self,
+        cache: BaseCache,
+        output_columns: Dict[str, Any],
+        file_path_column: str,
```

Comment:
simplify this to "url_column" or "input_column" - we don't need to support any local filepaths, just http/https urls. We'll download it, and if we cannot then return a `TransformError`